### PR TITLE
linked list started and readme written

### PR DESCRIPTION
**Pull request by wordhou**
- [X] Top-level README “Table of Contents” is updated
 - [X] Feature tasks for this challenge are completed
 - [X] Unit tests written and passing
     - [X] “Happy Path” - Expected outcome
     - [X] Expected failure
     - [X] Edge Case (if applicable/obvious)
 - [X] README for this challenge is complete
     - [X] Summary, Description, Approach & Efficiency, Solution
     - [X] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: challenges/lib/src/test/java/linkedList/LinkedListTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package linkedList;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import static java.util.Optional.ofNullable;
+
+public class LinkedListTest {
```
Comment (Mattpet26) [inline challenges/lib/src/test/java/linkedList/LinkedListTest.java:7]: Great work with your tests

File: challenges/lib/src/main/java/linkedList/LinkedList.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package linkedList;
+
+public class LinkedList<T> {
```
Comment (Mattpet26) [inline challenges/lib/src/main/java/linkedList/LinkedList.java:3]: Great work with your code, looks like you went with generic values!
